### DISCUSSION TITLE
placeholder api is crashing another plugin

### DISCUSSION THREAD

**Discussion by hyunsussi** [PlaceholderAPI/PlaceholderAPI#1019] [(2) Support] [answered]
It crashes [MoreFish] plugin and this is the log for it

server log - https://paste.helpch.at/jazaroriso.rb

papi dump - https://paste.helpch.at/uluvemobeg

Is there a way to solve this problem?

**Comment by Andre601** [comment, marked as answer]
MoreFish is using a no longer existing part of PAPI that has been removed for pretty much years now.

Just remove the plugin, or check if there is an updated version
